qe_metrics/libs/database.py

```diff
@@ -127,3 +134,44 @@ class JiraIssues(DB_CONNECTION.Entity):  # type: ignore
         customer_escaped = orm.Required(bool)
         date_created = orm.Required(date)
         last_updated = orm.Required(date)
+        date_record_modified = orm.Required(datetime, default=datetime.now)
+
+        @classmethod
+        def create_update_issues(
+            cls, issues: List[Issue], product: "Database.Products", severity: str, jira_server: str
+        ) -> None:
+            """
+            Create or update JiraIssues items in the database from a list of Jira issues.
+
+            Args:
+                issues (List[Issue]): A list of Jira issues
+                product (Database.Products): A product object
+                severity (str): Severity of the issues
+                jira_server (str): Jira server URL
+
+            Returns:
+                List["Database.JiraIssues"]: A list of JiraIssues objects
+            """
+            issues = [_issue for _issue in issues if _issue.fields.issuetype.name.lower() == "bug"]
+            for issue in issues:
+                existing_issue = cls.get(issue_key=issue.key, product=product)
+                if existing_issue:
+                    update_existing_issue(existing_issue, issue, severity)
```

Comment:
please use arg=val
motivation:
1. allows easier refactor (not using positional args)
2. more readable


please add this check to pre-commit:
- https://github.com/RedHatQE/openshift-python-wrapper/blob/main/.pre-commit-config.yaml#L18
- https://github.com/RedHatQE/openshift-python-wrapper/blob/main/.flake8#L52